### ISSUE TITLE
Set up Sentry error tracking

### ISSUE THREAD

**Issue by abettermap** [Language-Mapping/language-map#48]
## Description

Ross got the account in #15 and sent creds to Jason via email.

## Related items

#15

## Resolution

1. Follow Sentry tuts to set up "Project"
1. Start adding error handlers (at a minimum for `fetch` requests)

**Comment by abettermap** [comment]
@rperlin-ela I got this up and running. Setup for the basics was easy but there's A LOT to Sentry. First time using it so quite a plunge, but I really like the UI and capabilities.

I created a non-admin Sentry acct for myself so I can log in to see problems if needed. I don't anticipate fixing any outside today-ish since we've had a pretty good review window to catch the problems, but it will be handy to know what the problems are in order to document and address them. 

If you start getting pummeled by notifications at your ELA email let me know and we can dial it down. The ones you saw so far were a good indication that it's working, but one of them was just me testing and the other has been fixed.

I also ended up setting it to only accept events from the project's primary domain and (if i did it right) the Netlify deploys.

The free acct they gave us does not appear to have fancier things like performance monitoring and tracing, but the error monitoring is what we want so that was nice of them.

Like I said, lots more to it so I opened up #90. I don't know a ton but this is the first project (sadly) where I've ever gotten to use an error monitoring _service_ like this and I have to say it's very comforting to know when 💩  is breaking!
